app/views/layouts/_layout.njk

```diff
@@ -46,6 +46,21 @@
   </div>
 {% endblock %}
 
+{% block footer %}
+  {{ govukFooter({
```

Comment:
This can be simplified by using [items array syntax](https://design-system.service.gov.uk/components/footer/#footer-with-links-to-meta-information) e.g.
```
  {{ govukFooter({
    meta: {
      items: [
        {
          href: "/accessibility",
          text: "Accessibility"
        }
      ]
    }
  }) }}
```

Reply:
Thank you :)